metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same as the desktop file, this isn't upstreamed because the application is just an archive that you extract anywhere and run from there. It's not meant to be an application that's installed to your system, so we need to provide these.